common/formats/winexe_pe.cpp

```diff
@@ -49,6 +49,9 @@ void PEResources::clear() {
 bool PEResources::loadFromEXE(SeekableReadStream *stream, DisposeAfterUse::Flag disposeFileHandle) {
 	clear();
 
+	_exe = stream;
```

Comment:
The prefix for the commit log message should be "COMMON:", not "DIRECTOR:". See the affected files